src/Resources/Session.php

```diff
@@ -29,6 +29,18 @@ protected function handleGET()
      */
     protected function handlePOST()
     {
+         // IntegrateIo Hosted Trial Login
+         if ($this->getPayloadData('integrateio_id') !== null) {
```

Comment:
@tomonorman I'm not familiar with PHP but it seems that `!== null` is uncommon to be used at least in this repo. Can we use `if(!empty($this->getPayloadData('integrateio_id'))` here?

Reply:
indeed, thanks!